src/box/index.cc

```diff
@@ -554,6 +554,23 @@ index_build(struct index *index, struct index *pk)
 	return 0;
 }
 
+struct tuple *
+get_tuple_for_index_slow(struct index *index, struct tuple *tuple)
```

Comment:
Don't like the name of this function. We name function starting from the subject: index_get_tuple_slow().
Moreover, this function works as a predicate, so I'd refactor it this way:
 - make it return boolean (false, if tuple doesn't satisfy exclude_null condition, true otherwise)
 - rename to sort of index_tuple_is_excluded/filtered.

Reply:
It was a predicate in my first version. But then I need to do two different things: 1) evaluate predicate 2) change tuple to null if the predicate is true. If I do this within one function, then it complicates its behavior (see [comment](https://github.com/tarantool/tarantool/pull/5481#discussion_r535891800) from @alyapunov). If I set null value manually, then there will be a lot of repeating code in different places which I don't like. So I decided that the meaning of this function should be only smth like "return tuple which will be actually inserted in index". So no predicate evaluation is needed.

Reply:
why not `struct tuple* index_filter_tuple_slow(struct index *index, struct tuple *tuple)`?

Reply:
Why do you have to change tuple to null? Using current schema anyway two steps are required:
1) invoke get_tuple_for_index_slow()
2) return/skip tuple if the result is null
Using predicate:
1) invoke index_tuple_is_filtered()
2) return/skip tuple if the result is true